renderdoc/api/replay/replay_enums.h

```diff
@@ -733,6 +733,10 @@ DOCUMENT(R"(The dimensionality of a texture binding.
 .. data:: TextureCubeArray
 
   A Cubemap texture array.
+
+.. data:: TextureExternal
+
+  An External texture.
```

Comment:
External textures are not a type of texture, so there should not be a new enum in this list. External textures are normal textures (so one of the other types in this list) but with a different creation method that refers elsewhere outside the API. All APIs have external texture support in RenderDoc, including multiple existing extensions for importing external textures in GL, but they do not define a new texture type.